mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -248,18 +248,18 @@ class ALSModel private[ml] (
   @Since("1.3.0")
   def setPredictionCol(value: String): this.type = set(predictionCol, value)
 
+  private val predict = udf { (userFeatures: Seq[Float], itemFeatures: Seq[Float]) =>
```

Comment:
I think it's actually clearer to keep the names as it "instantiates" the usage to the reader.